RedirectApplication/RedirectApplication/Json/Json.cs

```diff
@@ -16,8 +16,13 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
 
         public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
         {
+            if (reader.Path == "TargetUrl")
```

Comment:
Example of JSON:

{
"TargetUrl":"localhost",
"Conditions": [ {"Name:":"ByBrowser","Browser":"Chrome","Url":"http://pikabu.ru/"} ]
}

Either we have just one field "TargetUrl", either we working with everything in "Conditions" (that may look like a tree).
I just don't know how to complicate this